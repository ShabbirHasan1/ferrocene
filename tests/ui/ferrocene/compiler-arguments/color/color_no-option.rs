// check-fail
// compile-flags: --error-format human --color

fn main() {}

// ferrocene-annotations: um_rustc_color
// ferrocene-annotations: um_rustc_error_format
