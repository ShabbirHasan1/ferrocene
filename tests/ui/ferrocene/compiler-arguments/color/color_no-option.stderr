error: Argument to option 'color' missing

